girder/utility/filesystem_assetstore_adapter.py

```diff
@@ -59,12 +59,13 @@ def validateInfo(doc):
         if not os.path.isabs(doc['root']):
             raise ValidationException('You must provide an absolute path '
                                       'for the root directory.', 'root')
-        if not os.path.isdir(doc['root']):
-            try:
-                os.makedirs(doc['root'])
-            except OSError:
-                raise ValidationException(
-                    'Could not make directory "%s".' % doc['root'])
+
+        try:
+            mkdir(doc['root'])
+        except Exception:
```

Comment:
Ideally, we would `except` only the exceptions that can happen in those cases. I actually am not a big fan of `except Exception`.